### PR TITLE
Dismiss search when going from Browse on iOS 10

### PR DESCRIPTION
**Pull request by yuliaveres**
The issue described here:
https://toshiapp.atlassian.net/browse/IOS-376

### CODE REVIEW

File: Toshi/Controllers/Browse/BrowseViewController.swift [hunk 1]
```diff
@@ -114,79 +112,78 @@ class BrowseViewController: SearchableCollectionController {
...
-
+        
         if let indexPathForSelectedRow = searchResultView.indexPathForSelectedRow {
             searchResultView.deselectRow(at: indexPathForSelectedRow, animated: true)
         }
+        
+        searchBar.text = nil
     }
-
+
```
Comment (yuliaveres) [inline Toshi/Controllers/Browse/BrowseViewController.swift]: can we get rid of all those white spaces ?

**Comment by yuliaveres** [comment]
Just need to hide cancel button after resigning first responder on search bar, and 👌 
and ✅

File: Toshi/Controllers/Browse/BrowseViewController.swift (hunk 1 above)
Comment (roberthein) [reply Toshi/Controllers/Browse/BrowseViewController.swift]: This is a problem throughout the app caused by *Swift Format* which removed the whitespace that Xcode automatically adds.

**Comment by roberthein** [comment]
Good catch bout that cancel button!

**Pull request merged**